uktrade/data-hub-frontend
# CDS-107 Filtering for State on the Companies Page

## Description of change

This PR adds a filter for US states to the company search page.

## Test instructions

Go to the company search page, in the sidebar you should see the filter 'US State' with a dropdown of US states, on selecting a state (such as Texas) the search results should only show relevant companies.

## Screenshots
### Before

![Screenie_7](https://user-images.githubusercontent.com/41294831/120303947-39dd0a80-c2c7-11eb-8323-8501f7f682af.png)


### After

![Screenie_6](https://user-images.githubusercontent.com/41294831/120303619-eb2f7080-c2c6-11eb-9e09-cbdd25a3ccc6.png)

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
